core/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -368,9 +372,9 @@ public void onFailure(Exception e) {
         protected ReplicationOperation<Request, ReplicaRequest, PrimaryResult<ReplicaRequest, Response>> createReplicatedOperation(
             Request request, ActionListener<PrimaryResult<ReplicaRequest, Response>> listener,
             PrimaryShardReference primaryShardReference, boolean executeOnReplicas) {
-            return new ReplicationOperation<>(request, primaryShardReference, listener,
-                executeOnReplicas, replicasProxy, clusterService::state, logger, actionName
-            );
+            return new ReplicationOperation<Request, ReplicaRequest, PrimaryResult<ReplicaRequest, Response>>(request,
```

Comment:
This was added unnecessarily, I've removed it in 5eafb1866e85b994fafd178d72c873bba9838e71